src/include/network/services/routing.rb

```diff
@@ -424,43 +438,65 @@ def validateRouting(_key, _event)
       false
     end
 
+    # Mangles route definition for storing in routes file
+    #
+    # Basically netmask field is obsolete, so it converts
+    # the record to use CIDR notation if netmask is defined.
+    #
+    # For details see "man routes".
+    def convert_route_conf(route)
+      return route if route["netmask"].empty? || route["netmask"] == "-"
+
+      dest = route["destination"]
+      netmask = route["netmask"]
+
+      if Netmask.Check4(netmask)
+        cidr = Netmask.ToBits(netmask)
+      elsif netmask.start_with?("/")
+        cidr = netmask[1..-1]
+      else
+        # if it is netmask then long netmask is not supported for IPv6
+        # if it is prefix length (CIDR), then prefix it has to be prefixed by '/'
+        raise ArgumentError, "Invalid netmask or prefix length"
```

Comment:
it is nice to add value to exception, so immediatelly visible like `"Invalid netmask or prefix length: '#{netmask}'"